dev-docs/howtoguides/testing.md

```diff
@@ -60,33 +60,29 @@ The testing can be overridden to install ubuntu-advantage-tools from other sourc
 To run the tests, you can use `tox`:
 
 ```shell
-tox -e behave-lxd-20.04
+tox -e behave
```

Comment:
This will run all of the tests sequentially, which is just about never desirable. Maybe add a warning about that here?